pkg/repositories/models/named_entity.go

```diff
@@ -15,6 +15,8 @@ type NamedEntityMetadataKey struct {
 // Fields to be composed into any named entity
 type NamedEntityMetadataFields struct {
 	Description string `gorm:"type:varchar(300)"`
+	// GORM doesn't save the zero value for ints, so we use a pointer for the State field
+	State *int32 `gorm:"default:0"`
```

Comment:
I didn't think about this before, but it seems like the type of thing we've handled before by not using `0` as a valid value for enums?

Reply:
this is the recommended approach for saving the default value, it's not particularly a hack to use a pointer other than it's slightly weird for golang